greenhouse_management/graphql/User.py

```diff
@@ -30,16 +33,29 @@ class CreateUser(graphene.Mutation):
 
     class Arguments:
         input = UserInput(required=True)
-        for_staff = graphene.Boolean()
 
     user = graphene.Field(UserType)
 
     @classmethod
-    def mutate(cls, root, info, input, for_staff=False):
-        if info.context.user.is_superuser and for_staff:
-            user = CustomUser.objects.create_superuser(**input)
+    def create_as_regular_user(cls, user_data):
+        permission_locked_fields = ['is_staff', 'is_superuser', 'is_active']
+        for field in permission_locked_fields:
+            user_data.pop(field, None)
+
+        user = CustomUser.objects.create_user(**user_data)
+        return user
+
+    @classmethod
+    def create_as_superuser(cls, user_data):
+        user = CustomUser.objects.create_superuser(**user_data)
```

Comment:
a nie bedzie sytuacji gdy superuser sprobuje utworzyc usera i poda mu np te parametry z UserInput na false i bedziemy miec superusera, ktory nie jest superuserem?
to juz lepiej z input fields wywalic te 3 pola bo jak wrzuci z false to manager wywali w tym miejscu:
```
        if extra_fields.get('is_staff') is not True:
            raise ValueError(_('Superuser must have is_staff=True.'))
        if extra_fields.get('is_superuser') is not True:
            raise ValueError(_('Superuser must have is_superuser=True.'))
```

Reply:
poprawione, zostawiłem te 3 pola aby dało się zrobić zwykłego usera z is_staff=True.